pkg/gatherers/clusterconfig/const.go

```diff
@@ -74,6 +74,15 @@ var (
 		Version:  "v1alpha1",
 		Resource: "clusterserviceversions",
 	}
+	oscpGroupVersionResource = schema.GroupVersionResource{
```

Comment:
should we also collect openstackversion Custom resources? /me not sure as i see controlplane/dataplane status field have containerImages and deployedVersion field populated so may be we will have duplicated data with openstackversion.

Reply:
After talking with @dprince I added 'openstackversion' CR to be collected too.